url78eec0e0b6_Orikuro_vic_tgz-pJ8-net_nexon_vindictus_itemcomparer_inport_StaticImportJ8/src/net/nexon/vindictus/itemcomparer/inport/StaticScrolls.java

```diff
@@ -0,0 +1,187 @@
+
+	private static Scroll_Suf sent = new Scroll_Suf("Sentinel", 224, -15, 0,
+			SCRPROP.getSenti(), HEAVY_PLATE_ALL);
+	private static Scroll_Suf resist = new Scroll_Suf("Resistant", 340, -160,
+			0, SCRPROP.getRes(), HEAVY_PLATE_ALL);
+	private static Scroll_Suf armadil = new Scroll_Suf(5, "Armadillo", 336, 0,
+			0, SCRPROP.getArmadi(), HEAVY_ALL);
+	private static Scroll_Suf master = new Scroll_Suf("Master", 0, 0, 0,
+			SCRPROP.getMaster(), -1, 5, 5, ONLY_ARMOR);
+	public final static Scroll_Suf enthu = new Scroll_Suf("Enthu", -370, 270,
+			282, SCRPROP.getEnthu(), 5, CLOTH_LIGHT_ALL);
+	private static Scroll_Suf force = new Scroll_Suf("Force", -28, 140, 132,
+			SCRPROP.getForce(), 2, CLOTH_LIGHT_ALL);
+
+	private static Scroll_Pre enli = new Scroll_Pre("Enlightenment", 0, 210,
+			210, 0, 0, 0, 0.0, CLOTH_LIGHT_PANTS_HELM, 2, 1, 1, 2, 35,
+			ItemTyp.CLOTH);
+	private static Scroll_Pre silent = new Scroll_Pre("Silent", 150, 110, 110,
+			0, 0, 0, 0.0, CLOTH_LIGHT_SHOES_GLOVES, 1, 2, -2, 2, 35,
+			ItemTyp.CLOTH);
+	private static Scroll_Suf journey = new Scroll_Suf("Journey", 150, 250,
+			250, 0, 0, 0, 0.0, CLOTH_LIGHT_PANTS_HELM, 1, 0, 3, 0, 35,
+			ItemTyp.CLOTH);
+	private static Scroll_Suf announc = new Scroll_Suf("Announcement", -330,
+			330, 330, 0, 0, 0, 0.0, CLOTH_LIGHT_SHOES_GLOVES, 0, 4, 1, 1, 35,
+			ItemTyp.CLOTH);
+
+	private static Scroll_Pre remember = new Scroll_Pre("Remember", 100, 0, 0,
+			0, 0, 0, 0.0, HEAVY_PLATE_PANTS_HELM, 2, 1, 2, 2, 35, ItemTyp.HEAVY);
+	private static Scroll_Pre repe = new Scroll_Pre("Repeatsay", 150, 0, 0, 0,
+			0, 0, 0.0, HEAVY_PLATE_SHOES_GLOVES, 1, 2, -1, 2, 35, ItemTyp.HEAVY);
+	private static Scroll_Suf exped = new Scroll_Suf("Expedition", 280, 170,
+			170, 0, 0, 0, 0.0, HEAVY_PLATE_PANTS_HELM, 0, 0, 4, 0, 35,
+			ItemTyp.HEAVY);
+	private static Scroll_Suf echo = new Scroll_Suf("Echo", -260, 190, 190, 0,
+			0, 0, 0.0, HEAVY_PLATE_SHOES_GLOVES, 0, 4, 2, 1, 35, ItemTyp.HEAVY);
+
+	private static Scroll_Pre time = new Scroll_Pre("Time", 160, 0, 0, 0, 0, 0,
+			0.0, ONLY_ARMOR, 4, -3, 6, 0, 0, ItemTyp.CLOTH);
+	private static Scroll_Suf stigma = new Scroll_Suf("Stigma", 70, 0, 0, 0, 0,
+			0, 0.0, ONLY_ARMOR, 5, -1, 7, 0, 0, ItemTyp.CLOTH);
+
+	public static List<Scroll_Pre> pres = new ArrayList<>();
+
+	public static List<Scroll_Suf> sufs = new ArrayList<>();
+
+	static {
+		pres.add(wb);
+		pres.add(tut);
+		pres.add(enli);
+		pres.add(silent);
+		pres.add(remember);
+		pres.add(repe);
+		pres.add(time);
+
+		sufs.add(force);
+		sufs.add(enthu);
+		sufs.add(sent);
+		sufs.add(armadil);
+		sufs.add(resist);
+		sufs.add(journey);
+		sufs.add(announc);
+		sufs.add(exped);
+		sufs.add(echo);
+		sufs.add(stigma);
+		sufs.add(master);
+
+	}
+
+	public static String[] prefix_Names() {
+
+		String[] values = new String[pres.size()];
+
+		for (int i = 0; i < pres.size(); i++) {
+			values[i] = pres.get(i).getName();
+		}
+		Arrays.sort(values);
+
+		return values;
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 148:155, 159:165.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.